packages/node/test/machine/integration/protocols2.spec.ts

```diff
@@ -120,9 +110,9 @@ describe("Three mininodes", () => {
         responderXpub: mininodeC.xpub,
         defaultTimeout: 100,
         initialState: appState,
-        initiatorBalanceDecrement: Zero,
-        responderBalanceDecrement: Zero,
-        tokenAddress: CONVENTION_FOR_ETH_TOKEN_ADDRESS,
+        initiatorBalanceDecrement: One,
+        responderBalanceDecrement: One,
+        tokenAddress: getAddress("88a5c2d9919e46f883eb62f7b8dd9d0cc45bc290"),
```

Comment:
1. You should not need to checksum a string literal in the code, instead either checksum it once and hard code it in the file.
2. Why magic constant? Perhaps instead use a dynamically generated string and put that in the test suite context?